scripts/find_files.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Copyright (C) 2022 Intel Corporation
+
+ SPDX-License-Identifier: MIT
+
+"""
+
+import os
+import sys
+import re
+
+"""
+    helper for obtaining all of the h/cpp files recursively,
+    starting from root_dir, with the exclusion of the build_dir
+"""
+
+def find_cpp_h_files_rec(root_dir, build_dir):
+    for path, dirs, files in os.walk(root_dir):
```

Comment:
`dirs` is unused. Use underscore `_` instead of this name for better readability:

` for path, _, files in os.walk(root_dir):`

Reply:
I've changed the approach and wrapped the ```add_executable/add_library/target_sources``` functions to automatically add ```cppformat``` target in cmake files so I've deleted this script for now.